packages/mobile/src/notifications/SummaryNotification.test.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react'
+import { Text, View } from 'react-native'
+import * as renderer from 'react-test-renderer'
+import SummaryNotification from 'src/notifications/SummaryNotification'
+
+const props = () => ({
+  image: require('src/images/placeholder.png'),
```

Comment:
Define the image path in /src/images and then just import the variable for use elsewhere